webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTableHelpers.js

```diff
@@ -69,6 +70,17 @@ export const groupSubscriptionsByProductId = ({ results: subscriptions }) => {
     grouped[subscription.product_id].subscriptions.unshift(subscription);
   });
 
+  for (product_id in grouped) {
```

Comment:
I see this code is changing the `groupSubscriptionsByProductId` function. I see what you're trying to do but I don't think this is the right place to do it. At this level of the code we don't want to affect the grouping since it's already correct.

Notice that this function is used here: https://github.com/Katello/katello/blob/3a485398c78a42ad02fb19989a5d09f919f147ea/webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTable.js#L33-L38

I see there is a function called 'buildTableRows' and I think that might be the better place to do this but I haven't looked at the implemtnation of that function.

Reply:
@jturel Since buildTableRows gets called every time we collapse, If I write the same logic inside buildTableRows it results in multiple generic rows. I'm trying to find a way around it if possible.

Reply:
@jturel I updated the code. Let me know if this looks good. I still need to figure out how to make entitlements "NA".